herddb-jdbc/src/main/java/herddb/jdbc/HerdDBDatabaseMetadata.java

```diff
@@ -947,7 +947,7 @@ public ResultSet getTableTypes() throws SQLException {
     @SuppressFBWarnings("SQL_NONCONSTANT_STRING_PASSED_TO_EXECUTE")
     public ResultSet getColumns(String catalog, String schemaPattern, String tableNamePattern, String columnNamePattern) throws SQLException {
 
-        String query = "SELECT table_name,column_name,data_type,auto_increment,is_nullable,ordinal_position FROM SYSCOLUMNS WHERE 1=1 ";
+        String query = "SELECT table_name,column_name,data_type,type_name,auto_increment,is_nullable,ordinal_position FROM SYSCOLUMNS WHERE 1=1 ";
```

Comment:
are we missing an UT for this change ?
I am expecting some assertion about DatabaseMetadata#getColumns